core/src/main/java/org/axonframework/config/EventHandlingConfiguration.java

```diff
@@ -678,19 +670,55 @@ public EventHandlingConfiguration configureListenerInvocationErrorHandler(
-     *                                              to use for the {@link org.axonframework.eventhandling.EventProcessor}
-     *                                              with the given {@code name}
+     *                                              to use for handler assigned to this group
      * @return this {@link EventHandlingConfiguration} instance for further configuration
      */
-    public EventHandlingConfiguration configureListenerInvocationErrorHandler(String name,
+    public EventHandlingConfiguration configureListenerInvocationErrorHandler(String processingGroup,
                                                                               Function<Configuration, ListenerInvocationErrorHandler> listenerInvocationErrorHandlerBuilder) {
-        listenerInvocationErrorHandlers.put(name, listenerInvocationErrorHandlerBuilder);
+        listenerInvocationErrorHandlers.put(processingGroup, listenerInvocationErrorHandlerBuilder);
+        return this;
+    }
+
+    /**
+     * Configures the Sequencing Policy to use when invoking event handlers asynchronously. The
+     * {@link SequencingPolicy}, defines which Event messages should be handled sequentially and which may be executed
+     * in parallel.
+     * <p>
+     * Note that the Sequencing Policy is generally ignored if both the Processor and the assigned EventHandlerInvoker
+     * are synchronous components (i.e. use the publishing thread to handle messages). This is the case when using a
+     * {@link SubscribingEventProcessor}, for example.
+     *
+     * @param sequencingPolicyBuilder The builder providing the default Sequencing Policy for this configuration
+     * @return this {@link EventHandlingConfiguration} instance for further configuration
+     */
+    public EventHandlingConfiguration configureSequencingPolicy(Function<Configuration, SequencingPolicy<? super EventMessage<?>>> sequencingPolicyBuilder) {
+        defaultSequencingPolicy.update(sequencingPolicyBuilder);
+        return this;
+    }
+
+    /**
+     * Configures the Sequencing Policy to use for handler assigned to the given {@code processingGroup}, when invoking
+     * these handlers asynchronously. The
+     * {@link SequencingPolicy}, defines which Event messages should be handled sequentially and which may be executed
+     * in parallel.
+     * <p>
+     * Note that the Sequencing Policy is generally ignored if both the Processor and the assigned EventHandlerInvoker
+     * are synchronous components (i.e. use the publishing thread to handle messages). This is the case when using a
+     * {@link SubscribingEventProcessor}, for example.
+     *
+     * @param processingGroup         The name of the processing group to configure this sequencing policy for
+     * @param sequencingPolicyBuilder The builder providing the default Sequencing Policy for this configuration
```

Comment:
This is not the default `SequencingPolicy` builder, but the `SequencingPolicy` builder for a specific `processingGroup`.